atproto/auth/oauth/oauth.go

```diff
@@ -242,11 +242,12 @@ func (cfg *ClientConfig) NewClientAssertion(authURL string) (string, error) {
 	}
 	claims := clientAssertionClaims{
 		RegisteredClaims: jwt.RegisteredClaims{
-			Issuer:   cfg.ClientID,
-			Subject:  cfg.ClientID,
-			Audience: []string{authURL},
-			ID:       secureRandomBase64(16),
-			IssuedAt: jwt.NewNumericDate(time.Now()),
+			Issuer:    cfg.ClientID,
+			Subject:   cfg.ClientID,
+			Audience:  []string{authURL},
+			ID:        secureRandomBase64(16),
+			IssuedAt:  jwt.NewNumericDate(time.Now()),
+			ExpiresAt: jwt.NewNumericDate(time.Now().Add(time.Minute * 1)),
```

Comment:
The duration here should definitely be a defined constant, not in-line.

There is an existing constant, `jwtExpirationDuration`, which is set to 30 seconds. If really necessary we could define a second constant, but I would just re-use that. I think the same security logic applies for both (aka, if we need more than 30 seconds for the client attestation, we probably do for request request tokens as well).